src/pages/Projects.jsx

```diff
@@ -8,7 +8,11 @@ const Projects = () => {
     // either make Hero reusable somehow (and move it to components/global)
```

Comment:
Can delete all the comments, now that you figured out direction.

Reply:
removed.